addons/snailmail_account/wizard/account_move_send.py

```diff
@@ -27,18 +29,30 @@ def _compute_checkbox_send_by_post(self):
             wizard.checkbox_send_by_post = not wizard.send_by_post_warning_message \
                 and wizard.company_id.invoice_is_snailmail
 
-    @api.depends('mode')
+    @api.depends('mode', 'checkbox_send_by_post')
```

Comment:
you are making cycling dependency here... `checkbox_send_by_post` depends of `send_by_post_warning_message` (see `_compute_checkbox_send_by_post`)...